lib/cfme/cloud_services/manifest_fetcher.rb

```diff
@@ -4,164 +4,26 @@ def self.fetch
     block_given? ? yield(manifest) : manifest
   end
 
+  def self.identity_headers
+    {"x-rh-identity" => Base64.strict_encode64(JSON.dump("identity" => {"account_number" => "cfme-tenant"}))}
```

Comment:
is "cfme-tenant" ok for manifest request or should I get somehow it from cfme subscription ?
thanks!

Reply:
If you're going through the external 3scale route this will have to be a valid user.  @abellotti was saying that there was a way to use the certificate from a registered appliance for authentication to 3scale hopefully he can help here.

Reply:
correct, if we're doing a new request to cloud.redhat.com, the x-rh-identity header will be ignore here as it's generated by the gateway.

If we're running on a valid insights client, we should be able to leverage the certs from the system to do cert-auth.  The url to hit for production needs to be cert.cloud.redhat.com.

So for this request:

```
https://cert.cloud.redhat.com/api/topological-inventory/cfme/manifest/5.11
```

Equivalent options for curl would be --cert and --key   for the cert.pem and key.pem,  ping me separately for the location of the certs.